src/sentry/deletions/__init__.py

```diff
@@ -24,6 +24,7 @@
 from __future__ import absolute_import
 
 from .base import BulkModelDeletionTask, ModelDeletionTask, ModelRelation  # NOQA
+from .defaults.group import GroupNodeDeletionTask  # NOQA
```

Comment:
Just for tests :(

Reply:
Could you please elaborate? What is the issue there ?

Reply:
This was the easiest way I could find to overwrite the DEFAULT_CHUNK_SIZE value, so that we would also be running through the batching logic in the test.

Reply:
Not following why you cannot just import GroupNodeDeletionTask in the test and overwrite?
But if this is a parameter you want to overwrite, why not making it an option ? It will be very hard to understand (later) why this import is here.

Reply:
Ah sorry, I think i messed up the import path before. Removed.